README.md

```diff
@@ -14,30 +14,183 @@ jobs:
 
     strategy:
       matrix:
-        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0"]
+        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0", "nightly"]
 
     steps:
-      - uses: actions/checkout@v4
-        with:
-          path: eox-hooks
-
-      - uses: eduNEXT/integration-test-in-tutor
+      - name: Run Integration Tests
+        uses: eduNEXT/integration-test-in-tutor@main
         with:
+          app_name: "eox-test"
           tutor_version: ${{ matrix.tutor_version }}
-          app_name: "eox-hooks"
-          shell_file_to_run: "eox_hooks/tests/tutor/integration.sh"
+          shell_file_to_run: "tests/integration.sh"
+          openedx_extra_pip_requirements: "package1==1.0 package2>=2.0"
+          fixtures_file: "fixtures/test_data.json"
+          openedx_imports_test_file_path: "tests/integration/import_tests.py"
+          openedx_imports_test_function_name: "test_openedx_imports"
 ```
 
 ## Inputs
 
 ### `app_name`
 
-**Required** Application name to test. E.g., eox-tenant.
+**Required**  
+The name of your plugin/application to test. This should match the directory name of your plugin.  
+*Example*: `"eox-tenant"`
 
 ### `tutor_version`
 
-**Required** The tutor version matrix to use.
+**Required**  
+The version of Tutor to use. You can specify:
+
+- A specific version number (e.g., `"==17.0.3"`).
+- A comparison operator with a version (e.g., `"<18.0.0"`).
+- The string `"nightly"` to use the latest development version.
+
+*Examples*:
+
+- `"==17.0.3"`
+- `"<18.0.0"`
+- `"nightly"`
```

Comment:
The same examples are shown above, could we use others, or leave it in one place?

Reply:
Done!